### PR TITLE
Dependabot addition

### PR DESCRIPTION
**Pull request by Quarks-1**
Added dependabot file to implement weekly dependency version checks for Node JS

changes in this PR:

adds a new file under `.github/` called `depndabot.yml`.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
+    directory: "/"
```
Comment (notmd) [inline .github/dependabot.yml:4]:
```suggestion
    directory: "/website"
```

**Review by notmd — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (Quarks-1) [reply .github/dependabot.yml]: Good catch, thanks!

**Comment by notmd** [comment]
@yk can you approve this? I'm not sure we need to invite the bot after the PR is merged or not. For private repo, it's required.

**Review by andreaskoepf — APPROVED** [review]
ok, let's try.

**Pull request merged**